api/src/main/java/org/pmiops/workbench/db/dao/InstitutionEmailDomainDao.java

```diff
@@ -0,0 +1,13 @@
+package org.pmiops.workbench.db.dao;
+
+import java.util.Set;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailDomain;
+import org.springframework.data.repository.CrudRepository;
+
+public interface InstitutionEmailDomainDao extends CrudRepository<DbInstitutionEmailDomain, Long> {
+
+  Set<DbInstitutionEmailDomain> getByInstitution(final DbInstitution institution);
+
+  long deleteByInstitution(final DbInstitution institution);
```

Comment:
How does the return value work? Do you get 0 if nothing is deleted or the ID if something was?

Reply:
a count of how many were deleted